core-bundle/src/Resources/contao/dca/tl_search_index.php

```diff
@@ -19,7 +19,8 @@
 			(
 				'id' => 'primary',
 				'pid' => 'index',
-				'word' => 'index'
+				'wordId' => 'index',
```

Comment:
Is the index on the `wordId` column really necessary? I think MySQL could also use the unique index over `wordId,pid` here.

Reply:
The wordId index should be way shorter I think, but I have to test this to see if it makes a difference.

Reply:
I just figured because we are using indexes that span multiple columns quite often:

https://github.com/contao/contao/blob/8e364e7574063a55f0db5d9842b94d6fd1f7bc8d/core-bundle/src/Resources/contao/dca/tl_page.php#L63-L69

Reply:
`'pid,type,start,stop,published'` seems wrong to me. The order would have to be `'pid,type,published,start,stop'` I think because start/stop are queried as ranges and not as exact values.

Reply:
It depends how you query. The order has to match the order in the `WHERE` statement.

Reply:
But I agree, the `published` should be filtered earlier. That would be way more efficient.

Reply:
Good point. We should check the other indexes as well and fix what needs to be fixed in a separate PR.

Reply:
See https://github.com/contao/contao/issues/1885